includes/rplus-wp-team-list.php

```diff
@@ -10,19 +10,20 @@
  * @package WP_Team_List
  */
 
-$role        = wp_team_list()->get_user_role( $user );
-$description = get_user_meta( $user->ID, 'description', true );
+$role_display_name = wp_team_list()->get_user_role( $user, 'display_name' );
+$role              = wp_team_list()->get_user_role( $user, 'name' );
+$description       = get_user_meta( $user->ID, 'description', true );
 ?>
-<div class="wp-team-member wp-team-list-item author-<?php echo esc_attr( $user->ID ); ?> role-<?php echo esc_attr( $role ); ?>">
+<div class="wp-team-member wp-team-list-item author-<?php echo esc_attr( $user->ID ); ?> <?php echo sanitize_html_class( 'role-' . $role ); ?>">
```

Comment:
@grappler Role names aren't sanitized at all. Because `add_role( 'foo bar', 'Foo Bar', [] );` is working we have to sanitize the value.

We could move the `sanitize_html_class()` call to another line:

```diff
 $role_display_name = wp_team_list()->get_user_role( $user, 'display_name' );
-$role              = wp_team_list()->get_user_role( $user, 'name' );
+$role_class        = sanitize_html_class( 'role-' . wp_team_list()->get_user_role( $user, 'name' ) );
 $description       = get_user_meta( $user->ID, 'description', true );
 ?>
-<div class="wp-team-member wp-team-list-item author-<?php echo esc_attr( $user->ID ); ?> <?php echo sanitize_html_class( 'role-' . $role ); ?>">
+<div class="wp-team-member wp-team-list-item author-<?php echo esc_attr( $user->ID ); ?> <?php echo esc_attr( $role_class ); ?>">
        <figure class="wp-team-member-avatar author-image">
                <?php echo wp_team_list()->get_avatar( $user ); ?>
```